salt/modules/bamboohr.py

```diff
@@ -75,10 +75,10 @@ def list_employees(order_by="id"):
         if cat.tag != "employees":
             continue
         for item in cat:
-            emp_id = item.items()[0][1]
+            emp_id = next(iter(item.items()))[1]
```

Comment:
`next(iter(item.values()))` does the same but 20% faster. Could you please update all the occurrences that use `items()` call?

Reply:
Did you mean `next(iter(item.items()))` in this case?

Reply:
great, done